x/distribution/client/rest/query.go

```diff
@@ -80,8 +81,19 @@ func delegatorRewardsHandlerFn(cliCtx context.CLIContext, cdc *codec.Codec,
 		if !ok {
 			return
 		}
+		var rewards types.QueryDelegatorTotalRewardsResponse
+		if err := cdc.UnmarshalJSON(res, &rewards); err != nil {
+			rest.WriteErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
 
-		rest.PostProcessResponse(w, cdc, res, cliCtx.Indent)
```

Comment:
We can leave `PostProcessResponse` if we remove the codec registration.

Reply:
Amended